superset-frontend/src/components/MetadataBar/MetadataBar.tsx

```diff
@@ -149,7 +149,10 @@ const Item = ({
     </StyledItem>
   );
   return isTruncated || collapsed || (tooltip && tooltip !== title) ? (
-    <Tooltip title={<TootipContent>{tooltip}</TootipContent>}>
+    <Tooltip
+      placement="bottom"
```

Comment:
@kgabryje I'm not sure if we should set this in the component itself. The original behavior follows the one defined in Ant Design that automatically repositions the tooltip depending on the available space and tooltip content. We may have cases where the user scrolls down and the tooltip will need to be displayed on top. If you want to ensure that in Explore the tooltip is always displayed at the bottom, then I suggest adding a property to the `MetadataBar` called `tooltipPlacement` and pass it to the Tooltip component.

Reply:
You should probably also add `autoAdjustOverflow` as a prop if you want to ensure the original positioning is respected independently if there's enough space available to display the tooltip. Keep in mind though, that setting this to `false` can lead to hidden content when displaying the tooltip.

Reply:
Good idea. I added prop `tooltipPlacement` to MetadataBar that defaults to `top` (just like antd's default) and set it to `bottom` in Explore and in drill to detail modal (because in the case of overflowing dataset name, it was also showing at the top while the rest of the tooltips were showing at the bottom)
<img width="730" alt="image" src="https://user-images.githubusercontent.com/15073128/198325742-6cc3337b-1db8-4001-a95c-50a5ee4fd30d.png">